engine/internal/pkg/discovery/agent.go

```diff
@@ -38,182 +36,160 @@ const (
 	EventTypeDel = EventType("del")
 )
 
+// NodeType describes the type of the node.
+type NodeType string
+
+const (
+	// NodeTypeMaster indicates that the node is a master.
+	NodeTypeMaster = NodeType("master")
+	// NodeTypeExecutor indicates that the node is an executor.
+	NodeTypeExecutor = NodeType("executor")
+)
```

Comment:
Why don't reuse `NodeType` defined in engine/model/cluster.go

Reply:
`NodeInfo` and `NodeType` defined in `engine/model/cluster.go` will be removed after we store executor info in MySQL.